### PR TITLE
simple cleanup

### PR DESCRIPTION
**Pull request by pca006132**

### CODE REVIEW

File: .github/workflows/manifold.yml [hunk 1]
```diff
@@ -27,7 +27,6 @@ jobs:
     env:
       CC: gcc-${{ matrix.gcc }}
       CXX: g++-${{ matrix.gcc }}
-    if: github.event.pull_request.draft == false
```
Comment (elalish) [inline .github/workflows/manifold.yml:30]: 👍

File: src/boolean3.cpp [hunk 2]
```diff
@@ -35,7 +35,7 @@ vec2 Interpolate(vec3 pL, vec3 pR, double x) {
   const bool useL = fabs(dxL) < fabs(dxR);
   const vec3 dLR = pR - pL;
   const double lambda = (useL ? dxL : dxR) / dLR.x;
-  if (!isfinite(lambda) || !isfinite(dLR.y) || !isfinite(dLR.z))
+  if (!std::isfinite(lambda) || !std::isfinite(dLR.y) || !std::isfinite(dLR.z))
```
Comment (elalish) [inline src/boolean3.cpp:38]: Looks good, but for my own info, what is the difference between e.g. `isfinite` and `std::isfinite`?

File: src/boolean_result.cpp [hunk 3]
```diff
@@ -14,9 +14,12 @@
 
 #include <algorithm>
 #include <array>
-#include <iostream>
 #include <map>
 
+#ifdef MANIFOLD_DEBUG
+#include <iostream>
```
Comment (elalish) [inline src/boolean_result.cpp:20]: Good catch! How long has that been there? Should we have this conditional include of string stuff in `utils.h` and then have a rule against including these headers anywhere else?

**Review by elalish — CHANGES_REQUESTED** [review]
Thanks, this looks great! How much of #969 does this address?

File: src/boolean3.cpp (hunk 2 above)
Comment (pca006132) [reply src/boolean3.cpp:38]: One is defined in cmath without namespace, one is in C++. I think they are practically the same.

File: src/boolean_result.cpp (hunk 3 above)
Comment (pca006132) [reply src/boolean_result.cpp:20]: Sounds good.

**Comment by pca006132** [comment]
Fixed 4 of them. Was focused on code cleanup, and removed a few files, so there are still trivial things that are not yet addressed here.

**Review by elalish — APPROVED** [review]
Thanks! Is this ready to merge?

**Pull request merged**